Library/Styles/ButtonStyles.swift

```diff
@@ -132,3 +132,23 @@ public let shareButtonStyle =
   <> UIButton.lens.image(for: .normal) .~ image(named: "icon--share")
   <> UIButton.lens.tintColor .~ .ksr_soft_black
   <> UIButton.lens.accessibilityLabel %~ { _ in Strings.dashboard_accessibility_label_share_project() }
+
+// MARK: - Read More Campaign Button
+
+public let controlReadMoreButtonStyle = baseButtonStyle
```

Comment:
I think we could separate the fact that the style is control/experimental from the naming of it, so I would call this something like `readMoreButtonTextStyle`.